InAppSettingsKitSampleApp/InAppSettingsKitSampleApp.xcodeproj/project.pbxproj

```diff
@@ -463,6 +471,7 @@
 				ALWAYS_SEARCH_USER_PATHS = NO;
 				CLANG_ENABLE_MODULES = YES;
 				COPY_PHASE_STRIP = YES;
+				DEVELOPMENT_TEAM = XTRCC2ZZE2;
```

Comment:
Whoops. That wasn't supposed to stay there. Sorry about that; I'll change it now.